src/datalad_installer.py

```diff
@@ -673,9 +673,19 @@ def main(self, argv: Optional[List[str]] = None) -> int:
             if not os.path.exists(path):
                 log.error("%s does not exist!", path)
                 ok = False
-            elif not os.access(path, os.X_OK):
+            elif not ON_WINDOWS and not os.access(path, os.X_OK):
                 log.error("%s is not executable!", path)
                 ok = False
+            else:
```

Comment:
```suggestion
            if ok:
```
so it is an additional check instead of an alternative one?

Reply:
It's an alternative check that's only run if the previous checks succeeded (since, if they fail, running `{cmd} --help` is unlikely to work).  I don't see the point of the suggested change.

Reply:
rright! I misread. thanks